packages/playground/src/components/pull-request/sections/pull-request-changes-section.tsx

```diff
@@ -237,43 +237,48 @@ const PullRequestChangesSection = ({
       </AccordionTrigger>
       <AccordionContent>
         <div>
-          {(minApproval && minReqLatestApproval && minApproval > minReqLatestApproval) ||
-            (!isEmpty(approvedEvaluations) && minReqLatestApproval === 0) ||
-            (minApproval && minApproval > 0 && minReqLatestApproval === undefined && (
-              <div className="ml-4">
-                <div className="flex pt-2 border-t mt-3 ml-2 items-center justify-between">
-                  {approvedEvaluations && minApproval <= approvedEvaluations?.length ? (
-                    <Text className="flex ml-2">
-                      <CheckCircleSolid className="text-success" />
-                      <Text className="pl-2 text-xs">
-                        {`Changes were approved by ${approvedEvaluations?.length} ${approvedEvaluations?.length === 1 ? 'reviewer' : 'reviewers'}`}
-                      </Text>
+          {((minApproval ?? 0) > (minReqLatestApproval ?? 0) ||
+            (!isEmpty(approvedEvaluations) && minReqLatestApproval === 0 && minApproval && minApproval > 0) ||
+            ((minApproval ?? 0) > 0 && minReqLatestApproval === undefined)) && (
+            <div className="ml-4">
+              <div className="flex pt-2 border-t mt-3 ml-2 items-center justify-between">
+                {approvedEvaluations && minApproval && minApproval <= approvedEvaluations?.length ? (
+                  <Text className="flex ml-2">
+                    <CheckCircleSolid className="text-success" />
+                    <Text className="pl-2 text-xs">
+                      {`Changes were approved by ${approvedEvaluations?.length} ${approvedEvaluations?.length === 1 ? 'reviewer' : 'reviewers'}`}
```

Comment:
Another candidate for `pluralize` function. If you don't want to implement, we can also use this light weight library https://www.npmjs.com/package/pluralize that would do the job for you.

Reply:
done